src/components/datepicker/datePicker.js

```diff
@@ -221,6 +250,9 @@
     /** @type {boolean} Whether the date-picker's calendar pane is open. */
     this.isCalendarOpen = false;
 
+    /** @final */
```

Comment:
I'm not quite sure what types of annotations these were. There were some other properties annotated that way.